### ISSUE TITLE
Menubar Links not Parsed Correctly

### ISSUE THREAD

**Issue by AntumDeluge** [doxygen/doxygen#9074]
I have the following in my `layout.xml`:

```xml
    <tab type="usergroup" title="Links">
      <tab type="user" visible="yes" title="Stendhal Homepage" url="https://stendhalgame.org/"/>
      <tab type="user" visible="yes" title="Arianne Homepage" url="https://arianne-project.org/"/>
      <tab type="user" visible="yes" title="Wiki" url="https://stendhalgame.org/wiki/Stendhal"/>
      <tab type="user" visible="yes" title="Sourceforge" url="https://sourceforge.net/projects/arianne"/>
      <tab type="user" visible="yes" title="GitHub" url="https://github.com/arianne/stendhal"/>
    </tab>
```

The URLs under the "Links" tab work correctly from the main page & some sub-pages. But from a class/namespace page, they do not.

- Tab link from `index.html`:
  - `https://github.com/arianne/stendhal`
- Tab link from `dc/d74/classgames_1_1stendhal_1_1common_1_1constants_1_1Actions.html`:
  - `file:///C:/Users/Jordan/Development/Arianne/stendhal/build/doxygen/html/https://github.com/arianne/stendhal`

**Version**
```
doxygen --version
1.9.3 (c0b9eafbfb53286ce31e75e2b6c976ee4d345473)
```

**Comment by albert-github** [comment]
It is a bit unclear what you have written in calsses / namespaces to appear on the class/namespace page, so:
- Can you please attach a, small, self contained example (source+configuration file in a tar or zip) that allows us to reproduce the problem? Please don't add external links as they might not be persistent.

**Comment by AntumDeluge** [comment]
Generated documentation is output to `build/doxygen`. Explanation is on main page.

[doxygen_sample_menubar_links.zip](https://github.com/doxygen/doxygen/files/7940612/doxygen_sample_menubar_links.zip)

**Comment by albert-github** [comment]
Minimal is a bit different, but I see the error here.

Looks like it has to do with the `CREATE_SUBDIRS=YES`

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #9077

**Comment by albert-github** [comment]
Code has been integrated in master on GitHub (please don't close the issue as this will be done at the moment of an official release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.9.4.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
